src/cloud/release-notes/mcp-release-notes.md

```diff
@@ -22,13 +22,17 @@ We have included all {{site.data.var.ce}} 2.x patches from the [Magento Technica
 
 This release includes the following updates:
 
--  {:.fix}<!--MAGECLOUD-4606-->**Updated patches to include all available patches for {{site.data.var.ce}} 2.x**–Updated the {{site.data.var.mcp}} package to include all {{site.data.var.ce}} 2.x patches available on the [Magento Download page](https://magento.com/tech-resources/download). If you copied any {{site.data.var.ce}} patches into your {{site.data.var.ece}} project previously, remove them to avoid conflicts.
+-  {:.fix}**Fix cron deadlocks and improve cron locking**–
 
--  {:.fix}<!--MAGECLOUD-4847-->**Updated patch for Elasticsearch catalog pagingation fix** –Replaced the Elasticsearch catalog pagination patch delivered in {{site.data.var.mcp}} v1.0 with a more effective fix.
+   -  {:.fix}Fixes an issue with some cron jobs not running due to an incorrect status value in the cron_schedule table. Now, we use the Magento lock framework to check and update cron job status instead of using the cron_schedule table. Cron jobs that have ended with an error status are retried during the next cron:run instead of waiting 24 hours.
```

Comment:
It looks like a command here, but if it is not executed on purpose by someone, I would just use the phrase "cron run" instead. If it is a manual run, then maybe define it like "the next time you initiate the `cron:run` command". Not sure what is most appropriate here, and it is NOT important enough stop the PR.

```suggestion
   -  {:.fix}Fixes an issue with some cron jobs not running due to an incorrect status value in the cron_schedule table. Now, we use the Magento lock framework to check and update cron job status instead of using the cron_schedule table. Cron jobs that have ended with an error status are retried during the next `cron:run` instead of waiting 24 hours.
```